tools/uvmboot/main.go

```diff
@@ -214,6 +219,9 @@ func main() {
 								logrus.Fatalf("Unrecognized value '%s' for option %s", c.String(outputHandlingArgName), outputHandlingArgName)
 							}
 						}
+						if pipe := c.String(consolePipeArgName); pipe != "" {
```

Comment:
Any reason we don't use the same `IsSet` pattern here as the rest of the args?

Reply:
Not particularly. It seemed silly to do two map lookups instead of one here, but probably consistency is more important.

Reply:
I agree it would be better if urfave/cli supported something like `if val, ok := c.String(name); ok {`.